### DISCUSSION TITLE
[Feature]: Panoramas: Use PhotoSphere Viewer to display photos with equirectangular projection

### DISCUSSION THREAD

**Discussion by CodeWithMa** [immich-app/immich#1630] [Feature Request]
### Feature detail

Maybe use one of these Libraries:

* https://photo-sphere-viewer.js.org/
* https://pannellum.org/

See this https://github.com/photoprism/photoprism/issues/352 for example photos.

### Platform

Web

**Comment by Vakrehus** [comment]
Oh yes, 360° images would be great. I use a lot of panoramic images which I would love to include in immich.

I think 360° support will be a very good feature.

**Comment by dmitry-brazhenko** [comment]
Also interested in 360° viewer

**Comment by dmitry-brazhenko** [comment]
Implemented a PR 

https://github.com/immich-app/immich/pull/3098

**Comment by ovizii** [comment]
Well, that PR was closed unmerged. Any news on 360 panorama support in immich?

**Reply by CodeWithMa** [reply]
It was merged in a different PR: https://github.com/immich-app/immich/pull/3412
